tests/test_archiver.py

```diff
@@ -65,20 +66,29 @@ def setUp(self):
             'input_biomaterial': biomaterials[0]
         }
 
+    def tearDown(self):
+        self.ingest_api.session.close()
+        self.usi_api.session.close()
+
     def _generate_fake_id(self, prefix):
         now = datetime.datetime.utcnow().strftime("%Y-%m-%dT%H%M%S")
         return prefix + '_' + now
 
     def test_get_archivable_entities(self):
         assay_bundle = self._mock_assay_bundle(self.bundle)
         self.archiver.get_assay_bundle = MagicMock(return_value=assay_bundle)
+        usi_api = MagicMock()
+        usi_api.url = 'usi_url'
+        usi_api.get_current_version = MagicMock(return_value=None)
+        self.archiver.usi_api = usi_api
```

Comment:
I think we'd be better off instantiating the archiver with a MagicMock() usi_api in the setUp() method. Would expect usi_api to be a private member and not accessible from outside like this, but python won't complain about that of course

Reply:
Good idea, have new commits to do this.